Tests/XCUITests/SaveLoginsTests.swift

```diff
@@ -23,6 +23,11 @@ class SaveLoginTest: BaseTestCase {
     private func saveLogin(givenUrl: String) {
         navigator.openURL(givenUrl)
         waitUntilPageLoad()
+        let browser = app.webViews["contentView"].otherElements["body"]
+        waitForExistence(browser.staticTexts["Username:"])
+        XCTAssertNotNil(browser.textFields.firstMatch.value)
+        waitForExistence(browser.staticTexts["Password:"])
+        XCTAssertNotNil(browser.secureTextFields.firstMatch.value)
```

Comment:
Here I ensured that the webpage is loaded with the username and the password shown.

The expected result of `testSaveLogin` is that two logins are saved:
![Simulator Screenshot - iPhone 14 - 2023-05-05 at 11 55 00](https://user-images.githubusercontent.com/1740517/236997928-c77e6d6d-88f2-4a52-8093-c24a16940f1a.png)

The actual result on bitrise is that only one login is saved with the username undefined.
![Screenshot_1_58B7871B-6679-4A6C-9B30-AC12B88F58A8](https://user-images.githubusercontent.com/1740517/236997979-45162d58-0c49-4c5e-a8c1-cc2fdf4cc509.jpeg)